tools/utils/code_change_tools.ts

```diff
@@ -1,17 +1,24 @@
-import {PORT, APP_DEST} from '../config';
+import {PORT, APP_DEST, APP_BASE, DIST_DIR} from '../config';
 import * as browserSync from 'browser-sync';
 
 let runServer = () => {
+  let baseDir = APP_DEST;
   let routes:any = {
-    [`/${APP_DEST}`]: APP_DEST,
-    '/node_modules': 'node_modules'
+    [`${APP_BASE}${APP_DEST}`]: APP_DEST,
+    [`${APP_BASE}node_modules`]: 'node_modules',
   };
+
+  if (APP_BASE !== '/') {
+    routes[`${APP_BASE}`] = APP_DEST;
+    baseDir = `${DIST_DIR}/empty/`;
```

Comment:
there might be a nicer way to do it, but as far as I can tell browserSync needs to have a baseDir so the options seem somewhat limited.